lib/standard/loads_runner.rb

```diff
@@ -1,9 +1,19 @@
+require_relative "runners/rubocop"
+require_relative "runners/version"
+require_relative "runners/verbose_version"
+require_relative "runners/help"
+
 module Standard
   class LoadsRunner
-    # Warning: clever metaprogramming. 99% of the time this is Runners::Rubocop
+    RUNNERS = {
+      rubocop: ::Standard::Runners::Rubocop,
+      version: ::Standard::Runners::Version,
+      verbose_version: ::Standard::Runners::VerboseVersion,
+      help: ::Standard::Runners::Help
+    }.freeze
+
     def call(command)
-      require_relative "runners/#{command}"
-      ::Standard::Runners.const_get(command.to_s.capitalize).new
```

Comment:
This meta-programming doesn't work with multiple-word command/constants (for example `:verbose_version`/`VerboseVersion`). Instead of modifying to account for this edge-case, I swapped out the code for a HashTable lookup. Less fun, I know. :(